core/connection.go

```diff
@@ -41,9 +48,9 @@ func CreateConnection(src, dst *ProvableChain, to time.Duration) error {
 		// In the case of success and this being the last transaction
 		// debug logging, log created connection and break
 		case connSteps.Success() && connSteps.Last:
-			log.Printf("★ Connection created: [%s]client{%s}conn{%s} -> [%s]client{%s}conn{%s}",
-				src.ChainID(), src.Path().ClientID, src.Path().ConnectionID,
-				dst.ChainID(), dst.Path().ClientID, dst.Path().ConnectionID)
+			logger.Info(
+				"★ Connection created",
```

Comment:
Is the lack of connection details deliberate?

Reply:
@bluele (cc @dongrie )
The logger create by `GetConnectionPairLogger()` automatically adds information of src/dst connections.